jdbc-driver/src/systemTest/java/com/google/cloud/dataproc/jdbc/DataprocSystemTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dataproc.jdbc;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.cloud.dataproc.v1beta2.ClusterControllerClient;
+import com.google.cloud.dataproc.v1beta2.ClusterControllerSettings;
+import com.google.common.collect.ImmutableMap;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class DataprocSystemTest {
+    private static final String REGION = System.getProperty("region");
+    private static final String PROJECT_ID = System.getProperty("projectId");
+    private static ClusterControllerClient clusterControllerClient;
+    private static Map<String, Map<String, String>> allLabelsMap;
+
+    @BeforeClass
+    public static void setUp() throws IOException, ExecutionException, InterruptedException {
```

Comment:
I think it's worth running cleanUp() at the beginning of setUp(), just in case clusters were leaked (not deleted) by previous runs. Leaked clusters are a big problem we have with our system tests in general